rails-snow-queen/app/assets/javascripts/quotingPanel.js

```diff
@@ -34,6 +34,7 @@ class QuotingPanel {
     this.polygons = polygons;
     this.totalAreaInSqFt = this.convertToSqFt(this.aggregateAreaInMts());
     this.totalDue = this.calculateTotalDue();
+    this.showTotalsNode();
```

Comment:
probably negligible delay,  but you should probably show the node _after_ you've updated the Area and Total

Reply:
done